react/src/pages/data/bulk/ExportV2.tsx

```diff
@@ -0,0 +1,229 @@
+import Box from '@mui/material/Box';
+import DataTable from 'components/table/DataTable';
+import { CritterStrings, CritterStrings as CS, MapStrings } from 'constants/strings';
+import { RowSelectedProvider } from 'contexts/TableRowSelectContext';
+import { useTelemetryApi } from 'hooks/useTelemetryApi';
+import EditCritter from 'pages/data/animals/EditCritter';
+import ExportViewer from 'pages/data/bulk/ExportImportViewer';
+import AddEditViewer from 'pages/data/common/AddEditViewer';
+import ManageLayout from 'pages/layouts/ManageLayout';
+import { SyntheticEvent, useEffect, useState } from 'react';
+import { Animal, AttachedAnimal } from 'types/animal';
+import { QueryStatus } from 'react-query';
+import { columnToHeader, doNothing, doNothingAsync, headerToColumn } from 'utils/common_helpers';
+import { SpeciesProvider } from 'contexts/SpeciesContext';
+import DateInput from 'components/form/Date';
+import dayjs from 'dayjs';
+import { InboundObj, parseFormChangeResult } from 'types/form_types';
+import Select from 'components/form/BasicSelect';
+import { Button } from 'components/common';
+import { Grid, Tab, Tabs, Typography } from '@mui/material';
+import { CreateFormField } from 'components/form/create_form_components';
+import { wfFields } from 'types/events/event';
+import { BCTWFormat } from 'types/common_types';
+import SelectCode from 'components/form/SelectCode';
+import { formatDay } from 'utils/time';
+
+type ColumnOptions = 'species' | 'population_unit' | 'wlh_id' | 'animal_id' | 'device_id' | 'frequency';
+
+export interface IFormRowEntry {
+    column: ColumnOptions;
+    operator: string;
+    value: string | number;
+}
+
+class PossibleColumnValues implements BCTWFormat<PossibleColumnValues> {
+    species: string;
```

Comment:
I dont think this needs to be its own class. Reuse Animal / Collar classes for dropdowns